datasets/Economy.md

```diff
@@ -50,6 +60,11 @@ Wage and salary data for Indian states.
 #### [Mexico Census](https://en.www.inegi.org.mx/temas/)
 Population Census and Statistics for Mexico at Country and State levels.
 
+### [National Institution for Transforming India.](https://niti.gov.in/)
+
+#### [India Poverty Status](https://www.niti.gov.in/sites/default/files/2021-11/National_MPI_India-11242021.pdf)
+India poverty statistics - percentage of people below poverty line.
```

Comment:
Maybe we want to remove "percentage of people below poverty line" in case we add more / different variables from this source later?